lib/puppet/type/jail.rb

```diff
@@ -45,6 +45,27 @@
     desc 'Hostname of the jail'
   end
 
+  newproperty(:pcpu) do
+    desc "Cap the CPU usage of a jail"
```

Comment:
I think just single quotes on the `desc` lines will get the tests to pass our ruby style checker.

Reply:
Changed it over.

Reply:
Looks like it's passing now :)